azure-pipelines.yml

```diff
@@ -148,9 +164,12 @@ stages:
           export MACOSX_DEPLOYMENT_TARGET=10.15
           export PATH=/usr/local/opt/flex/bin:/usr/local/opt/bison/bin:$PATH
           export SDKROOT=$(xcrun --sdk macosx --show-sdk-path)
-          export NRN_BUILD_FOR_UPLOAD=1
-          sudo mkdir -p /opt/nrnwheel/$(uname -m)
-          sudo tar -zxf $(readlineSF.secureFilePath) --directory /opt/nrnwheel/$(uname -m)
+          # export NRN_BUILD_FOR_UPLOAD=1
+          export ARCHTYPE=$(arch) # arm64 or x86_64 or universal2
```

Comment:
The ARCHTYPE provenance between bash scripts, cmake, and setup.py is confusing. In .github/workflows/neuron-ci.yml it is not specified but left to packaging/python/build_wheels.bash to determine the reasonable ARCHTYPE. i.e.  ```uname -m``` on the build machine (for cmake, the CMAKE_HOST_SYSTEM_PROCESSOR).